hooks/contracts/kleros.ts

```diff
@@ -0,0 +1 @@
+export const KLEROS_ARBITRATOR_ADDRESS = '0xe40DD83a262da3f56976038F1554Fe541Fa75ecd';
```

Comment:
can't we use constants file for this? I would not create a file just for this constant.

Reply:
Also inside hooks, doesn't make sense for me.

Reply:
Since it's something contract related, I fill like it's on a good spot. It's related to contracts and it's related to kleros, so `contracts/kleros` makes sense for me. Having a bunch of different data inside the same constants folder, not sure if it's the best approach, since it's difficult to know what constants are inside that file. This way I can go through the folder structure and easily find data related to kleros contracts. What do you think about this?

Reply:
it seems a bit confusing, but will let you decide, we can always change later if doesn't stick.

But why inside hooks folder?

Reply:
Because it's contract related stuff. But will move it from here